### PR TITLE
Jcsp 776 maintenance and improvements

### PR DESCRIPTION
**Pull request by anastasiiagnes**
Splitting ViewBoard into ViewScrumBoard, ViewKanbanBoard, ViewNextGenBoard and ViewBacklog to achieve more flexibility with the load profiles

### CODE REVIEW

File: custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewNextGenBoard.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    viewBoardMeasureType = MeasureType.VIEW_NEXT_GEN_BOARD,
+    issuePreviewMeasureType = MeasureType.ISSUE_PREVIEW_NEXT_GEN_BOARD,
+    contextOperationMeasureType = MeasureType.CONTEXT_OPERATION_NEXT_GEN_BOARD,
+    configureBoardMeasureType = MeasureType.CONFIGURE_NEXT_GEN_BOARD,
+    viewIssueProbability = viewIssueProbability,
+    configureBoardProbability = configureBoardProbability,
+    contextOperationProbability = contextOperationProbability), Action {
+
+    override fun run() {
+        val board = getBoard(nextGenBoardsMemory as SeededMemory<BoardPage>)
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewNextGenBoard.kt]: this one can be extracted to the viewBoardContent method too - just need to specify a parameter for the memory

File: custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    private val measure: Measure,
+    private val issueKeyMemory: IssueKeyMemory,
+    private val viewBoardMeasureType: ActionType<Unit>,
+    private val issuePreviewMeasureType: ActionType<Unit>,
+    private val contextOperationMeasureType: ActionType<Unit>,
+    private val configureBoardMeasureType: ActionType<Unit>,
+    private val viewIssueProbability: Float,
+    private val configureBoardProbability: Float,
+    private val contextOperationProbability: Float
+) {
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt]:
```suggestion
) : Action {
```

File: custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        measure.measure(MeasureType.CONFIGURE_BOARD, configureBoardProbability) {
+            measure.measure(measureType, isSilent = false) {
+                board
+                    .configureBoard()
+                    .waitForLoadPage()
+            }
+        }
+    }
+
+    protected fun viewBoardContent(board: BoardPage) {
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt]: Let's rename this method - it doesn't make only view board
e.g. `generalRun`

File: src/test/kotlin/jces1209/AwsVus.kt [hunk 4]
```diff
@@ -63,7 +61,7 @@ class AwsVus(
             ).provision()
         }
         val provisioned = MulticastVirtualUsersFormula.Builder(
-                nodes = (configProperties?.getProperty("setting.nodes"))?.toInt() ?: 6,
+                nodes = 6,
```
Comment (ysemenkov) [inline src/test/kotlin/jces1209/AwsVus.kt]: typo?

**Review by ysemenkov — CHANGES_REQUESTED** [review]

File: src/test/kotlin/jces1209/AwsVus.kt (hunk 4 above)
Comment (anastasiiagnes) [reply src/test/kotlin/jces1209/AwsVus.kt]: corrected

File: custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt (hunk 3 above)
Comment (anastasiiagnes) [reply custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewBoard.kt]: What about `runBoardActions`?

File: custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewKanbanBoard.kt [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    configureBoardMeasureType = MeasureType.CONFIGURE_KANBAN_BOARD,
+    viewIssueProbability = viewIssueProbability,
+    configureBoardProbability = configureBoardProbability,
+    contextOperationProbability = contextOperationProbability) {
+
+    override fun run() {
+        val board = runBoardActions(kanbanBoardsMemory as SeededMemory<BoardPage>, "Kanban")
+
+        if (board != null) {
+        repeat(2) {
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/action/boards/ViewKanbanBoard.kt]: formatting
```suggestion
             repeat(2) {
```

**Review by ysemenkov — APPROVED** [review]

**Pull request merged**